packages/Webkul/Category/src/Repositories/CategoryRepository.php

```diff
@@ -294,6 +292,25 @@ public function getPartial($columns = null)
         return $trimmed;
     }
 
+    /**
+     * Get searched categories.
+     *
+     * @return void
+     */
+    public function getSearchedCategories(array $params = [])
+    {
+        $queryBuilder = $this->query()
+            ->select('categories.*')
+            ->leftJoin('category_translations', function ($join) {
+                $join->on('categories.id', '=', 'category_translations.category_id')
+                    ->where('category_translations.locale', app()->getLocale());
+            })
+            ->where('category_translations.name', 'like', '%'.urldecode($params['query']).'%')
+            ->orderBy('created_at', 'desc');
+
+        return $queryBuilder->paginate($params['limit'] ?? 10);
+    }
```

Comment:
Can you please try to use the getAll method of this repo and reduce this method.

Reply:
Done.